priv/graphql/schemas/admin/session.gql

```diff
@@ -0,0 +1,81 @@
+"""
+Allow admin to get information about sessions.
+"""
+type SessionAdminQuery @protected{
+  "Get the list of established sessions for a specified domain or global"
```

Comment:
```suggestion
  "Get the list of established sessions for a specified domain or globally"
```
I think it's grammatically incorrect - an adverb is needed here. Same would apply below.